### DISCUSSION TITLE
Self-nomination for Contributor role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#99] [Content Team]
Hi @paketo-buildpacks/content-maintainers 

I would like to self-nominate for the contributor role for the content team. I have added documentation for various Python features (Poetry, SBOM), and I intend on actively contributing further documentation for upcoming features (e.g. Ruby SBOM, any changes to the dependencies track).

Thanks,
Rob

**Comment by robdimsdale** [comment]
@TisVictress @fg-j can I get another 👍 so we have super majority please?

**Comment by robdimsdale** [comment]
@paketo-buildpacks/steering-committee looks like we have a super majority for me to be a member of the content contributors team.

**Comment by dmikusa** [comment]
Congrats! @robdimsdale I've added you to the contributor's team.
